### ISSUE TITLE
Implement ANN with softmax

### ISSUE THREAD

**Issue by guustavov** [guustavov/myms#6]
This issue refers to the implementation of a neural network using softmax in the output layer. The goal is to get rid of the intermediate range (used to define which instances need to be reclassified by KNN algorithm). Knowing that softmax sets a probability for each possible label of the element to be classified, a threshold of the output of the network could play this role.
